src/main/webapp/js/util.js

```diff
@@ -79,6 +79,47 @@ function createDealCard(deal) {
   return dealCard;
 }
 
+/**
+ * Returns a container for a deal.
+ * @param {object} deal deal whose info will be shown.
+ * @param {number} numCol the bootstrap width of each deal
+ * @param {number} sectionId the sectionId of each deal
+ * @return {HTMLDivElement} a DOM element showing deal's info.
+ */
+function createHomeDealCard(deal, numCol, sectionId) {
+  const dealBody = document.createElement('div');
+  dealBody.classList.add('col-md-'+numCol, 'mt-5');
+  const date = new Date(Date.parse(deal.timestamp));
+  dealBody.innerHTML += `
+      <div id=deal-card-${sectionId} class="card deal-card h-100">
+        <img class="card-img-top home-deal-img" src=${deal.pic} alt="">
+        <div class="card-body d-flex flex-column">
+        <div class="card-text float-right"><i class="fa fa-thumbs-up"></i>  
+          ${deal.votes}</div>
+          <div class="card-text">${date.toLocaleString()}</div>
+          <div class="d-flex justify-content-end"></div>
+          <h5 class="card-title">${deal.description}</h5>
```

Comment:
If I post a deal with description `<button onclick="alert(1)">xss</button>`, there are some security problems
![Screenshot 2020-07-29 at 10 43 19 AM](https://user-images.githubusercontent.com/39252823/88750538-5466c480-d188-11ea-899b-90721a6e57fc.png)
same for the other fields

Reply:
i realised i also forgot to escape html somewhere, you can copy the util function from #139

Reply:
Ill make a separate PR for this? After #139 gets merged

Reply:
ok